lib/eslint.js

```diff
@@ -264,6 +264,11 @@ function modifyConfigsFromComments(ast, config, reportingConfig) {
                         enableReporting(reportingConfig, comment.loc.start, Object.keys(parseListConfig(value)));
                         break;
 
+                    case "eslint-ignore":
+                        disableReporting(reportingConfig, { "line": comment.loc.start.line, "column": 0 }, Object.keys(parseListConfig(value)));
+                        enableReporting(reportingConfig, comment.loc.end, Object.keys(parseListConfig(value)));
```

Comment:
This would allow you to ignore multiple lines.
Example: 
```js
var a = eval('alert()'), /* eslint-ignore no-eval




*/ b = eval('alert()');
```